pkg/apis/v1beta1/crd_validation_cel_test.go

```diff
@@ -675,3 +845,13 @@ var _ = Describe("CEL/Validation", func() {
 		})
 	})
 })
+
+var _ = AfterEach(func() {
+	// Clean up all AKSNodeClasses created during LocalDNS tests
```

Comment:
nit: it is customary to place global After* handlers in front of the file. Also, this will cleanup _all_ AKSNodeCLasses (not just those created during LocalDNS tests) - which is fine and desired